pkg/cmd/experimental/syncgroups/config/types.go

```diff
@@ -0,0 +1,148 @@
+package config
```

Comment:
I disagree here, I don't think these query templates and annotation/label keys (only used by the sync, not used at all by the ldap password auth system) should remain in this place. I've changed the package they're in but I have not moved them to the ldappassword package.